regression/pages/studio/privacy_policy.py

```diff
@@ -0,0 +1,15 @@
+"""
+Privacy Policy page
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.studio import BASE_URL
+
+
+class PrivacyPolicy(PageObject):
+    """
+    Terms of Service page
+    """
+    url = BASE_URL + '/edx-privacy-policy'
+
+    def is_browser_on_page(self):
+        return 'edX Privacy Policy' in self.browser.title
```

Comment:
You're right that this is a flaky condition. I dont' know what that page looks like, but I assume there's some privacy policy text? Could you define `is_browser_on_page` on whether or not that particular CSS element is visible?